course3/homework-vector/vector.cc

```diff
@@ -3,9 +3,59 @@
 
 class Vector
 {
-    // TODO
```

Comment:
Huh, I think this must be a hang over from when I started doing all the assignments in master... those are now in the new branch I created. I'll see what I can do about that for the next time.